draft-ietf-cats-framework.md

```diff
@@ -134,87 +134,87 @@ Furthermore, this document describes a workflow of the main CATS procedures (see
 
 CATS Service ID (CS-ID):
  : An identifier representing a service, which the clients use to access it. See {{cats-ids}}.
 
 Service instance:
   : An instance of running resources according to a given service logic.
-  : Many such instances can be enabled by a service provider. Instances that adhere to the same service logic provide the same service.
-  : An instance is running in a service site. Clients’ requests are serviced by one or more of these instances.
+  : A service provider may enable many service instances that adhere to the same service logic to provide the same service.
+  : A service instance runs in a service site and one or more instances may service clients’ requests.
 
 Service site:
- : A location that hosts the resources that are required to offer a service.
+ : A location that hosts the resources that implement an offered service.
  : A service site may be a node or a set of nodes.
- : A CATS-serviced site is a service site that is connected to a CATS-Forwarder.
+ : A CATS-serviced site is a service site that a service provider connects to a CATS-Forwarder.
 
 Service contact instance:
   : A client-facing service function instance that is responsible for receiving requests in the context of a given service.
   : A service contact instance can handle one or more service instances.
   : Steering beyond a service contact instance is hidden to both clients and CATS components.
-  : A service request is processed according to the service logic (e.g., handle locally or solicit backend resources).
+  : A service contact instance processes a client's service request according to the service logic (e.g., handle locally or solicit backend resources).
   : A service contact instance is reachable via at least one Egress CATS-Forwarder.
-  : A service can be accessed via multiple service contact instances running at the same or different locations (service sites).
+  : Clients may access a service via multiple service contact instances running at the same or different locations (service sites).
   : A service contact instance may dispatch service requests to one or more service instances (e.g., a service contact instance that behaves as a service load-balancer).
 
 CATS Service Contact Instance ID (CSCI-ID):
  : An identifier of a specific service contact instance. See {{cats-ids}}.
 
 Service request:
- : A request to access or invoke a specific service. Such a request is steered to a service contact instance via CATS-Forwarders.
- : A service request is placed using service-specific protocols.
- : Service requests are not explicitly sent by clients to CATS-Forwarders.
+ : A request to access or invoke a specific service. CATS-Forwarders steer a service request to a service contact instance.
+ : Clients must place a service request using service-specific protocols.
+ : Clients direct service requests to a CS-ID without explicit knowledge of CATS-Forwarders.
```

Comment:
to a CS-ID? This sentence highlight that the destination is the service provider address, not the CATS forwarder.

Clients direct service requests to a server (identified by a CS-ID),  without explicit knowledge of CATS-Forwarders.
may be better?